README.md

```diff
@@ -63,178 +24,93 @@ For more information on the importance of a professional README for your reposit
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [First-Project] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[First-Project]** is a html/css project that I have done to teach me how to use linters.
```

Comment:
- [ ] Kindly consider providing a descriptive name title of the project in the readme. This is to ensure that the readme is professional and repository visitor can have a quick understanding of what the project is about. At most cases, the title may be same as the repository name ☺️.

Reply:
Thank you, I will edit that.